pages/reviewer/ReviewerHandler.php

```diff
@@ -64,79 +58,35 @@ public function authorize($request, &$args, $roleAssignments)
     {
         $context = $request->getContext();
         if ($context->getData('reviewerAccessKeysEnabled')) {
-            $this->_validateAccessKey($request);
+            $accessKeyCode = $request->getUserVar('key');
+            if ($accessKeyCode) {
+                $url = $request->getDispatcher()->url(
+                    $request,
+                    PKPApplication::ROUTE_PAGE,
+                    $request->getContext()->getPath(),
+                    PKPInvitationHandler::REPLY_PAGE,
+                    PKPInvitationHandler::REPLY_OP_ACCEPT,
+                    null,
+                    [
+                        'key' => $accessKeyCode,
+                    ]
+                );
+
+                header('HTTP/1.1 301 Moved Permanently');
+                $request->redirectUrl($url);
+                return true;
```

Comment:
This is dead code; `PKPRequest::redirect` [stops execution](https://github.com/pkp/pkp-lib/blob/93e4a76fe2018e05d0830b677c43294017fb8724/classes/core/PKPRequest.php#L128).

Reply:
Sure, removed